packages/web-react/src/components/Modal/README.md

```diff
@@ -1,52 +1,24 @@
 # Modal
 
-```jsx
-import { Modal, ModalDialog, ModalBody, ModalHeader, ModalFooter, Button } from '@lmc-eu/spirit-web-react/components';
-```
-
-```jsx
-const [isOpen, setOpen] = useState(false);
-
-const toggleModal = () => setOpen(!isOpen);
+This is React implementation of the [Modal] component.
 
-const handleClose = () => {
-  setOpen(false);
-};
+Modal is a composition of several subcomponents:
 
-// Trigger
-<Button onClick={toggleModal} aria-expanded={isOpen} aria-controls="#ModalExample">
-  {isOpen ? 'Close' : 'Open'} Modal
-</Button>
+- [Modal](#modal)
+  - [ModalDialog](#modaldialog)
+    - [ModalHeader](#modalheader)
+    - [ModalBody](#modalfooter)
+    - [ModalFooter](#modalfooter)
 
-// Modal
-<Modal id="ModalExample" isOpen={isOpen} onClose={handleClose}>
-  <ModalDialog>
-    <ModalHeader>Modal Title</ModalHeader>
-    <ModalBody>Modal Body</ModalBody>
-    <ModalFooter description="ModalFooter description">
-      <Button color="primary" onClick={handleClose}>
-        Confirm
-      </Button>
-      <Button color="tertiary" onClick={handleClose}>
-        Cancel
-      </Button>
-    </ModalFooter>
-  </ModalDialog>
-</Modal>
-```
+## Modal
 
-## An example using a form as a ModalDialog element
+Modal establishes the top layer with backdrop. Under the hood it uses the [`<dialog>` element][mdn-dialog] which
```

Comment:
The link to the `mdn-dialog` is missing.
![Screenshot 2023-09-14 at 13 38 23](https://github.com/lmc-eu/spirit-design-system/assets/2481010/4b19aae6-d274-4339-927a-49a30c2eaf86)

Reply:
Fixed.